Microsoft.Toolkit.Uwp.UI.Controls/MarkdownTextBlock/MarkdownTextBlock.cs

```diff
@@ -1271,13 +1275,27 @@ public void RegisterNewHyperLink(Hyperlink newHyperlink, string linkUrl)
             try
             {
                 return eventArgs.Handled
-                                ? eventArgs.Image
-                                : new BitmapImage(new Uri(url));
+                    ? eventArgs.Image
+                    : GetImageSource(url);
             }
             catch (Exception)
             {
                 return null;
             }
+
+            ImageSource GetImageSource(string imageUrl)
+            {
+                if (IsSvgImageSupported)
+                {
+                    if (imageUrl.EndsWith(".svg"))
```

Comment:
Make this case insensitive.

Could use `Extension` if using `Uri`.

Reply:
@michael-hawker not sure what is `Extension`. We can use `Path.GetExtension` here.

Reply:
For a `Uri` instance, there is a property called `Extension`.

`Path.GetExtension()` will throw an exception with Urls

Reply:
@WilliamABradley Sorry, I don't see Extension property neither in docs (https://docs.microsoft.com/en-us/dotnet/api/system.uri?view=netframework-4.7.1) nor in intellisense. And Path.GetExtension() does not throw on absolute path of Uri for me.